docs/data-sources/ip_ranges.md

```diff
@@ -1,9 +1,5 @@
 ---
-layout: "fastly"
 page_title: "Fastly: fastly_ip_ranges"
-sidebar_current: "docs-fastly-datasource-ip_ranges"
-description: |-
```

Comment:
The new frontmatter doesn't support these legacy fields:
https://www.terraform.io/docs/registry/providers/docs.html#yaml-frontmatter

Reply:
UPDATE: `tfplugindocs` dynamically added `description: |-` to the index.md so it would appear there's more frontmatter options than what is actually _documented_.

Reply:
sure I'll put back all frontmatter for now.